pkg/ansible/watches/watches.go

```diff
@@ -240,16 +260,10 @@ func verifyGVK(gvk schema.GroupVersionKind) error {
 func verifyAnsiblePath(playbook string, role string) error {
 	switch {
 	case playbook != "":
-		if !filepath.IsAbs(playbook) {
-			return fmt.Errorf("playbook path must be absolute")
-		}
 		if _, err := os.Stat(playbook); err != nil {
 			return fmt.Errorf("playbook: %v was not found", playbook)
 		}
 	case role != "":
-		if !filepath.IsAbs(role) {
-			return fmt.Errorf("role path must be absolute")
-		}
 		if _, err := os.Stat(role); err != nil {
```

Comment:
Is this check working with just the role name?

Reply:
probably not. I pushed the wip changes to check with you folks if I got properly the suggestion.